diff --git a/node_modules/@vuepress/theme-default/assets/404.png b/node_modules/@vuepress/theme-default/assets/404.png
new file mode 100644
index 0000000..3d8e230
Binary files /dev/null and b/node_modules/@vuepress/theme-default/assets/404.png differ
diff --git a/node_modules/@vuepress/theme-default/assets/404_cloud.png b/node_modules/@vuepress/theme-default/assets/404_cloud.png
new file mode 100644
index 0000000..c6281d0
Binary files /dev/null and b/node_modules/@vuepress/theme-default/assets/404_cloud.png differ
diff --git a/node_modules/@vuepress/theme-default/layouts/404.vue b/node_modules/@vuepress/theme-default/layouts/404.vue
index 2cbfa0f..e11a52a 100644
--- a/node_modules/@vuepress/theme-default/layouts/404.vue
+++ b/node_modules/@vuepress/theme-default/layouts/404.vue
@@ -1,30 +1,254 @@
-<template>
-  <div class="theme-container">
-    <div class="theme-default-content">
-      <h1>404</h1>
-
-      <blockquote>{{ getMsg() }}</blockquote>
+<script>
+export default {
+  data() {
+	return {
+	  message: "阿噢，一不小心来到了荒无人烟的沙漠"
+	};
+  }
+};
+</script>
 
-      <RouterLink to="/">
-        Take me home.
-      </RouterLink>
-    </div>
+<template>
+  <div class="wscn-http404-container">
+	<div class="wscn-http404">
+	  <div class="pic-404">
+		<img
+		  class="pic-404__parent"
+		  src="../assets/404.png"
+		  alt="404"
+		/>
+		<img
+		  class="pic-404__child left"
+		  src="../assets/404_cloud.png"
+		  alt="404"
+		/>
+		<img
+		  class="pic-404__child mid"
+		  src="../assets/404_cloud.png"
+		  alt="404"
+		/>
+		<img
+		  class="pic-404__child right"
+		  src="../assets/404_cloud.png"
+		  alt="404"
+		/>
+	  </div>
+	  <div class="bullshit">
+		<div class="bullshit__oops">OOPS!</div>
+		<div class="bullshit__info">
+		  All rights reserved
+		  <a
+			style="color: #20a0ff"
+			href="https://vuepress.vuejs.org/zh/theme/option-api.html#globallayout"
+			target="_blank"
+		  >vuepress</a
+		  >
+		</div>
+		<div class="bullshit__headline">{{ message }}</div>
+		<div class="bullshit__info">
+		  请检查您输入的网址是否正确，或返回首页。
+		</div>
+		<a href="/" class="bullshit__return-home">回首页</a>
+	  </div>
+	</div>
   </div>
 </template>
 
-<script>
-const msgs = [
-  `There's nothing here.`,
-  `How did we get here?`,
-  `That's a Four-Oh-Four.`,
-  `Looks like we've got some broken links.`
-]
+<style lang="stylus" scoped>
 
-export default {
-  methods: {
-    getMsg () {
-      return msgs[Math.floor(Math.random() * msgs.length)]
-    }
+.wscn-http404-container {
+  position: relative !important;
+}
+
+.wscn-http404 {
+  position: absolute !important;
+  top: 50% !important;
+  transform: translateY(50%) !important;
+  width: 1200px;
+  padding: 0 50px;
+  overflow: hidden;
+}
+
+.wscn-http404 > .pic-404 {
+  position: relative;
+  float: left;
+  width: 600px;
+  overflow: hidden;
+}
+
+.pic-404__parent {
+  width: 100%;
+}
+
+.pic-404__child {
+  position: absolute;
+}
+
+
+.left {
+  top: 17px;
+  left: 220px;
+  width: 80px;
+  opacity: 0;
+  animation-name: cloudLeft;
+  animation-duration: 2s;
+  animation-timing-function: linear;
+  animation-delay: 1s;
+  animation-fill-mode: forwards;
+}
+
+.mid {
+  top: 10px;
+  left: 420px;
+  width: 46px;
+  opacity: 0;
+  animation-name: cloudMid;
+  animation-duration: 2s;
+  animation-timing-function: linear;
+  animation-delay: 1.2s;
+  animation-fill-mode: forwards;
+}
+
+.right {
+  top: 100px;
+  left: 500px;
+  width: 62px;
+  opacity: 0;
+  animation-name: cloudRight;
+  animation-duration: 2s;
+  animation-timing-function: linear;
+  animation-delay: 1s;
+  animation-fill-mode: forwards;
+}
+
+@keyframes cloudLeft {
+  0% {
+	top: 17px;
+	left: 220px;
+	opacity: 0;
+  }
+  
+  20% {
+	top: 33px;
+	left: 188px;
+	opacity: 1;
+  }
+  
+  80% {
+	top: 81px;
+	left: 92px;
+	opacity: 1;
+  }
+  
+  100% {
+	top: 97px;
+	left: 60px;
+	opacity: 0;
   }
 }
-</script>
+
+@keyframes cloudRight {
+  0% {
+	top: 100px;
+	left: 500px;
+	opacity: 0;
+  }
+  
+  20% {
+	top: 120px;
+	left: 460px;
+	opacity: 1;
+  }
+  
+  80% {
+	top: 180px;
+	left: 340px;
+	opacity: 1;
+  }
+  
+  100% {
+	top: 200px;
+	left: 300px;
+	opacity: 0;
+  }
+}
+
+.bullshit {
+  position: relative;
+  float: left;
+  width: 300px;
+  padding: 30px 0;
+  overflow: hidden;
+}
+
+.bullshit__oops {
+  margin-bottom: 20px;
+  font-size: 32px;
+  font-weight: bold;
+  line-height: 40px;
+  color: #1482f0;
+  opacity: 0;
+  animation-name: slideUp;
+  animation-duration: 0.5s;
+  animation-fill-mode: forwards;
+}
+
+.bullshit__headline {
+  margin-bottom: 10px;
+  font-size: 20px;
+  font-weight: bold;
+  line-height: 24px;
+  color: #222;
+  opacity: 0;
+  animation-name: slideUp;
+  animation-duration: 0.5s;
+  animation-delay: 0.1s;
+  animation-fill-mode: forwards;
+}
+
+.bullshit__info {
+  margin-bottom: 30px;
+  font-size: 13px;
+  line-height: 21px;
+  color: grey;
+  opacity: 0;
+  animation-name: slideUp;
+  animation-duration: 0.5s;
+  animation-delay: 0.2s;
+  animation-fill-mode: forwards;
+}
+
+.bullshit__return-home {
+  display: block;
+  float: left;
+  width: 110px;
+  height: 36px;
+  font-size: 14px;
+  line-height: 36px;
+  color: #fff;
+  text-align: center;
+  cursor: pointer;
+  background: #1482f0;
+  border-radius: 100px;
+  opacity: 0;
+  animation-name: slideUp;
+  animation-duration: 0.5s;
+  animation-delay: 0.3s;
+  animation-fill-mode: forwards;
+}
+
+@keyframes slideUp {
+  0% {
+	opacity: 0;
+	transform: translateY(60px);
+  }
+  
+  100% {
+	opacity: 1;
+	transform: translateY(0);
+  }
+}
+
+
+</style>
+
