diff --git a/node_modules/vue-router/src/util/scroll.js b/node_modules/vue-router/src/util/scroll.js
index d1adeb3..eaa5e85 100644
--- a/node_modules/vue-router/src/util/scroll.js
+++ b/node_modules/vue-router/src/util/scroll.js
@@ -141,9 +141,10 @@ function scrollToPosition (shouldScroll, position) {
   if (isObject && typeof shouldScroll.selector === 'string') {
     // getElementById would still fail if the selector contains a more complicated query like #main[data-attr]
     // but at the same time, it doesn't make much sense to select an element with an id and an extra selector
+    const escapedSelector = CSS.escape(shouldScroll.selector);
     const el = hashStartsWithNumberRE.test(shouldScroll.selector) // $flow-disable-line
       ? document.getElementById(shouldScroll.selector.slice(1)) // $flow-disable-line
-      : document.querySelector(shouldScroll.selector)
+      : document.querySelector(escapedSelector)
 
     if (el) {
       let offset =
