diff --git a/node_modules/vuepress-plugin-smooth-scroll/lib/enhanceApp.js b/node_modules/vuepress-plugin-smooth-scroll/lib/enhanceApp.js
index fbe861a..3ca56dd 100644
--- a/node_modules/vuepress-plugin-smooth-scroll/lib/enhanceApp.js
+++ b/node_modules/vuepress-plugin-smooth-scroll/lib/enhanceApp.js
@@ -1,4 +1,5 @@
 import '../styles/index.styl';
+import { hashRE } from 'vuepress-theme-vdoing/util';
 export default ({ Vue, router }) => {
     router.options.scrollBehavior = (to, from, savedPosition) => {
         if (savedPosition) {
@@ -11,7 +12,12 @@ export default ({ Vue, router }) => {
             if (Vue.$vuepress.$get('disableScrollBehavior')) {
                 return false;
             }
-            const targetElement = document.querySelector(to.hash);
+            // 解码并清理 to.hash 值以去除特殊字符
+            const decodedHash = decodeURIComponent(to.hash);
+            const sanitizedHash = decodedHash.replace(/[^\w-]/g, '');
+
+            const targetElement = document.querySelector('#' + sanitizedHash);
+           // const targetElement = document.querySelector(to.hash);
             if (targetElement) {
                 return window.scrollTo({
                     top: getElementPosition(targetElement).y,
